### PR TITLE
react-form-controls - James Yang

### PR DESCRIPTION
**Pull request by jamesdae**
https://user-images.githubusercontent.com/110082070/208021401-9a1e9dd1-b4e9-4744-bd76-6500e6825dbd.mov

### CODE REVIEW

File: react-form-controls/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      <form action="">
+        <label>
+          Username
+          <input value={this.state.username} onChange={this.handleUsernameChange} type="text" />
+        </label>
+        <label>
+          Password
+          <input value={this.state.password} onChange={this.handlePasswordChange} type="password" />
+        </label>
+        <button onClick={this.handleSubmit}>Sign Up</button>
```
Comment (Zhiwutian) [inline react-form-controls/src/index.jsx:40]: this should be on the form itself and be `onSubmit` just like in html forms

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
make sure to add onSubmit to the form

**Comment by jamesdae** [comment]
https://user-images.githubusercontent.com/110082070/208162714-4b2f22b8-42e8-4eed-a68d-6d9071f3d22c.mov

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**